src/main/java/policycompass/fcmmanager/controllers/FCMModels.java

```diff
@@ -753,4 +758,81 @@ public static FCMModel LoadData() {
 		session.close();
 		return model;
 	}
+
+	public static int isAdminUser(String userPath,String userToken)
+	{
+		int returnValue=-1;
+		String charset = "UTF-8";
+		String requestURL = "https://adhocracy-frontend-stage.policycompass.eu/api";
+		String requestURLGod = "https://adhocracy-frontend-stage.policycompass.eu/api/principals/groups/gods/";
+
+		try {
+
+			URL url = new URL(requestURL);
+			HttpURLConnection httpConn = (HttpURLConnection) url.openConnection();
+			httpConn.setUseCaches(false);
+			httpConn.setDoOutput(true); // indicates POST method
+			httpConn.setDoInput(true);
+
+			httpConn.setRequestProperty("X-User-Path", userPath);
+			httpConn.setRequestProperty("X-User-Token", userToken);
+
+			List<String> response = new ArrayList<String>();
+			int status = httpConn.getResponseCode();
+			if (status == HttpURLConnection.HTTP_OK) {
+				BufferedReader reader = new BufferedReader(new InputStreamReader(
+						httpConn.getInputStream()));
+				String line = null;
+				while ((line = reader.readLine()) != null) {
+					response.add(line);
+				}
+				reader.close();
+				httpConn.disconnect();
+
+				if(response.toString().toLowerCase().contains("error")){
+					returnValue=-1;
+					throw new IOException("Invalid User." );
+				}
+				else{
+					returnValue=0;
+
+					url = new URL(requestURLGod);
+					httpConn = (HttpURLConnection) url.openConnection();
+					httpConn.setUseCaches(false);
+					httpConn.setDoOutput(true); // indicates POST method
+					httpConn.setDoInput(true);
+
+					httpConn.setRequestProperty("X-User-Path", userPath);
+					httpConn.setRequestProperty("X-User-Token", userToken);
+					response = new ArrayList<String>();
+					status = httpConn.getResponseCode();
+					if (status == HttpURLConnection.HTTP_OK) {
+						reader = new BufferedReader(new InputStreamReader(
+								httpConn.getInputStream()));
+						line = null;
+						while ((line = reader.readLine()) != null) {
+							response.add(line);
+						}
+						reader.close();
+						httpConn.disconnect();
+						if(response.toString().toLowerCase().contains("error")){
+							returnValue=0;
+						}
+						else{
+							returnValue=1;
```

Comment:
Simply going from the Adhocracy docs, this would mean that every user is classified as god.
You don't check whether a user is actually part of the 'god' list.